patches/react-router-dom+6.2.1.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/react-router-dom/index.d.ts b/node_modules/react-router-dom/index.d.ts
+index 146c97d..27ff56f 100644
+--- a/node_modules/react-router-dom/index.d.ts
++++ b/node_modules/react-router-dom/index.d.ts
+@@ -74,7 +74,8 @@ export declare const NavLink: React.ForwardRefExoticComponent<NavLinkProps & Rea
+  * use in our exported `<Link>`.
+  */
+ export declare function useLinkClickHandler<E extends Element = HTMLAnchorElement>(to: To, { target, replace: replaceProp, state }?: {
+-    target?: React.HTMLAttributeAnchorTarget;
++  // @ts-ignore
++  target?: React.HTMLAttributeAnchorTarget;
```

Comment:
Build could not find the type of `React.HTMLAttributeAnchorTarget`. Fixing this required updating types of react, but when I did that, I got bunch of other errors.